cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofileinstance.json

```diff
@@ -0,0 +1,41 @@
+{
+  "time-series-profile":
+  {
+    "location-id": "DET",
+    "key-parameter": "Depth",
+    "parameter-list": [ "Temp-Water", "Pres", "%-DO",  "Conc-DO", "Cond", "pH", "TurbN",  "Depth","Volt-Battery"],
+    "description": "Demonstration Profile for Couger Lake",
+    "office-id": "SWT",
+    "ref-ts-id": "DET.Elev-Forebay.Inst.0.0.Mixed-Rev"
+  },
+
+  "value-list" :  [
+    {
+      "time-zone": "US/Pacific",
+      "times" : ["09/09/2019,13:16:01", "09/09/2019,13:17:20"],
+      "value":
+      {
+        "parameter": "Temp-Water",
+        "unit": "F",
+        "values": ["1", "2"]
+      }
+    },
+    {
+      "value":
```

Comment:
Times should be either within each "value" or outside the value list.

Reply:
when they are part of the value, does that mean the profile timeseries can have different time arrays?

Reply:
I don't believe so but we generally try to avoid any usage side ambiguity with CDA input/output.

Reply:
value-list, maybe a better name would be timeseries-list. I had the times inside the list because the timearray corresponds to all value arrays.

Reply:
Oh, I didn't see the nesting correctly. It's okay at that level. When I first looked at it I thought the times were only in the first "value" which mean the user always has to parse the first one if they want it or not.

Reply:
changed values to be time-value pairs